pyproject.toml

```
@@ -1,23 +1,182 @@
 [build-system]
-requires = ["setuptools>=46.1.0", "setuptools_scm[toml]>=5", "wheel"]
+requires = ["setuptools>=65.4.1", "setuptools_scm[toml]>=7", "wheel"]
 build-backend = "setuptools.build_meta"
 
+
+[project]
+name = "iqm-cortex_cli"
```

```suggestion
name = "iqm-cortex-cli"
```
For consistency, package names are normalized: https://packaging.python.org/en/latest/specifications/name-normalization/